### PR TITLE
Inline component fixes

### PR DESCRIPTION
**Pull request by marcialfps**
### Changes
- Fix gap value when showing the divider.
- Fix divider height when using alignY.
- Changes in story examples.

### CODE REVIEW

File: lib/src/inline/Inline.stories.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export const Chromatic = () => (
     </Container>
     <Title title="AlignX = end" theme="light" level={4} />
```
Comment (Lisa159) [inline lib/src/inline/Inline.stories.tsx:104]: Add `with divider` in the title.

File: lib/src/inline/Inline.stories.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const Chromatic = () => (
     </Container>
     <Title title="AlignY = end" theme="light" level={4} />
```
Comment (Lisa159) [inline lib/src/inline/Inline.stories.tsx:57]: Add `with divider` in the title.

File: lib/src/inline/Inline.stories.tsx (hunk 1 above)
Comment (marcialfps) [reply lib/src/inline/Inline.stories.tsx]: Done!

File: lib/src/inline/Inline.stories.tsx (hunk 2 above)
Comment (marcialfps) [reply lib/src/inline/Inline.stories.tsx]: Done!

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**